KIPs/kip-81.md

```diff
@@ -0,0 +1,122 @@
+Each GC member can appoint a voter account that will cast votes on behalf of the KLAY in its CnStakingContracts. The voter account is necessary because CnStakingContract itself cannot send transactions to cast votes.
+
+![gc voting structure](../assets/kip-81/gc_voting_structure.png)
+  
+Staked balances and voter accounts are tracked in a new special smart contract named StakingTracker. StakingTracker stores the voter list and their voting powers for each voting period. CnStakingContract must notify StakingTracker to recognize any balance changes. Below is an example of StakingTracker contract interface.
+```solidity
+interface IStakingTracker {
+
+  struct Tracker {
+    // this struct is updated only if trackStart <= block.number <= trackEnd.
+    uint64 trackStart;
+    uint64 trackEnd;
+
+    // List of eligible nodes and their staking contracts.
+    // Determined at createTracker() and won’t change.
+    address[] nodeIds;
+    mapping(address => address) stakingToNodeId;
+
+
+    // Balances and voting powers of each nodes.
+    // Updated by notifyStake() and notifyWithdraw().
+    mapping(address => uint256) nodeBalances;
+    mapping(address => uint256) votingPowers;
+    uint256 totalVotes;
+
+
+    // Appointed voters list
+    // Updated by appointVoter().
+    mapping(address => address) nodeIdToVoter;
+    mapping(address => address) voterToNodeId;
+  }
+
+  // Called by Voting at proposal submission, to finalize voter list.
+  function createTracker(uint256 trackEnd) public returns (uint trackerId);
+
+
+  // Called by CnStakingContract whenever its balance changes.
+  function notifyStake(uint256 amount) public;
+  function notifyWithdraw(uint256 amount) public;
+
+  /// Called by Governance admin or CnStaking admins to change voter account.
+  function appointVoter(address nodeId, address voter) public;
+
+  /// Called by Voting to calculate voting outcome.
+  function getVotingPower(uint trackerId, address voter)
+    public view returns(uint256);
+  function quorums(uint trackerId)
+    public view returns(uint quorumCount, uint quorumPower); 
+}
+```
+
+## Expected Effect
+The proposed GC Voting method is expected to produce the following changes:
+- All members in the Klaytn ecosystem grow together with credibility 
+- Entrusting obligation and authority to GC members motivates active participation in governance voting activities and KLAY staking 
+- Anyone can easily view the proposals and voting status through the governance portal. It encourages holders to participate and give responsibility and authority to the GC members.
+- The Klaytn network can take a step closer to transparency and decentralization.
+ 
+## Backward Compatibility
+n/a
```

Comment:
Is this compatible with previous versions? anything breaks compatibility?

Reply:
There is no change in Klaytn node in this KIP. It's only on-chain activity.

The GovParam contract and related Klaytn Kore hardfork is out of scope.

Reply:
@kjhman21 Added the backward compatibility section.

There is no change in Klaytn core. But GCs should deploy a new CnStakingV2 contract to participate in on-chain voting.